### PR TITLE
[11.0] [MIG] account_due_list

### PR DESCRIPTION
**Pull request by ozono**
Migration to V 11.0

### CODE REVIEW

File: account_due_list/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Usage
+=====
+
+To use this module, you need to go to:
+
+* Invoicing > Adviser > Payments and due list
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/96/10.0
```
Comment (angelmoya) [inline account_due_list/README.rst]: 11.0

File: account_due_list/views/payment_view.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data>
```
Comment (angelmoya) [inline account_due_list/views/payment_view.xml]: data tag is not necesary

**Review by angelmoya — APPROVED** [review]

File: account_due_list/README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (HviorForgeFlow) [inline account_due_list/README.rst]: change to https url

File: account_due_list/README.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (HviorForgeFlow) [inline account_due_list/README.rst]: change to .png

**Comment by pedrobaeza** [comment]
@hveficent instead of making single comments, you can start a new review and add all the comments to be posted at once.

File: account_due_list/README.rst (hunk 4 above)
Comment (ozono) [reply account_due_list/README.rst]: I have changed it... but in already migrated modules in this repo is .svg and I can't find any related annotation in *Migration-to-version-11.0*

**Pull request merged**